storage/rocksdb/ha_rocksdb.cc

```diff
@@ -292,6 +292,42 @@ static int rocksdb_force_flush_memtable_now(
   return HA_EXIT_SUCCESS;
 }
 
+static void rocksdb_force_flush_memtable_and_lzero_now_stub(
+    THD *const thd, struct st_mysql_sys_var *const var, void *const var_ptr,
+    const void *const save) {}
+
+static int rocksdb_force_flush_memtable_and_lzero_now(
+    THD *const thd, struct st_mysql_sys_var *const var, void *const var_ptr,
+    struct st_mysql_value *const value) {
+  sql_print_information("RocksDB: Manual memtable and L0 flush\n");
+  rocksdb_flush_all_memtables();
+
+  const Rdb_cf_manager &cf_manager = rdb_get_cf_manager();
+  for (const auto &cf_handle : cf_manager.get_all_cf()) {
+    rocksdb::ColumnFamilyMetaData metadata;
+    rdb->GetColumnFamilyMetaData(cf_handle, &metadata);
```

Comment:
I tested a few things, like passing in an SST file that no longer exists, or trying to compact a level that doesnt exist, and in both cases `CompactFiles` handled it gracefully.


It'll write something to the logs like so:
```
2017-03-20 18:26:50 2306114 [Warning] MyRocks: failed to read/write in RocksDB. Error type = RDB_IO_ERROR_GENERAL, stat
us code = 4, status = Invalid argument: Specified compaction input file /000099.sst does not exist in column family def
ault.
```